docs/manual-instrumentation.md

```diff
@@ -0,0 +1,45 @@
+# Manually instrument a .NET application
+
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches, you'll be
+able to present a more detailed representation of the logic and functionality
+of your application, clients, and framework.
+
+To create your custom traces you should execute following steps.
+
+1. Add the `System.Diagnostics.DiagnosticSource` dependency to your project:
+
+    ```xml
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+    ```
+
+1. Create `ActivitySource` instance
```

Comment:
```suggestion
2. Create an `ActivitySource` instance:
```

Reply:
You can always put 1. in numbered list. It is easier to add points in the middle of the list. It will be handled correctly by md-displayer.